mobile/examples/React_Native/ort-rn-basic-usage/App.tsx

```diff
@@ -0,0 +1,69 @@
+import { StatusBar } from 'expo-status-bar';
+import { Alert, Button, StyleSheet, Text, View } from 'react-native';
+
+import * as ort from 'onnxruntime-react-native';
+import { Asset } from 'expo-asset';
+
+let myModel: ort.InferenceSession;
+
+async function loadModel() {
+  try {
+    // TODO: Can also add example code to demonstrate loading model from bytes here
+    const assets = await Asset.loadAsync(require('./assets/mnist.ort'));
+    const modelUri = assets[0].localUri;
+    if (!modelUri) {
+      Alert.alert('failed to get model URI', `${assets[0]}`);
+    } else {
+      myModel = await ort.InferenceSession.create(modelUri);
+      Alert.alert(
+        'model loaded successfully',
+        `input names: ${myModel.inputNames}, output names: ${myModel.outputNames}`);
+    }
+  } catch (e) {
+    Alert.alert('failed to load model', `${e}`);
+    throw e;
+  }
+}
+
+async function runModel() {
+  try {
+    // Prepare model input data
+    const inputData = new Float32Array(28 * 28);
```

Comment:
Add a comment saying in real usage you must set the inputData to the actual input values.